uwuno/src/main/java/com/learning/uwuno/room.java

```diff
@@ -26,14 +28,19 @@
     private deck deck;
     private Status roomStatus;
     private player playerTurn;
+    private gameSettings gameSettings;
+    private gameState gameState;
 
     // Class functions
+    // TODO: remove useBlankCards from param (useBlankCards moved to gameSettings)
```

Comment:
It's not a double entry. 

I removed it, but there may be some null pointer risks since a few things in this file references deck. I added deck creation in `setupPlayerAndDeck`.